dd-sdk-android-ndk/src/main/cpp/utils/signal-monitor.c

```diff
@@ -121,9 +122,14 @@ void handle_signal(int signum, siginfo_t *info, void *user_context) {
     for (int i = 0; i < signals_array_size; i++) {
         const int signal = handled_signals[i].signal_value;
         if (signal == signum) {
-            crash_signal_intercepted(signal,
-                                     handled_signals[i].signal_name,
-                                     handled_signals[i].signal_error_message);
+            static const size_t backtrace_max_size = 8 * 1024; // 8KB
```

Comment:
Where does this limit come from? Is it unified with iOS?

Reply:
It is not...as I agreed with Simon it is big enough to make sure it gets all the stacktrace.

Reply:
I just checked with them and they do not actually have a limit for the stack trace ...they just go backwards until they hit `main`. On my end I followed what others were doing and I am limiting the stack size to `30` lines. I might have to do what they are doing and compute the stack memory size while processing it and get rid of this magic number here if I can.